app/views/namespaces/components/_form.html.slim

```diff
@@ -10,9 +10,7 @@ div ref="form" class="collapse"
           span v-if="!$v.namespace.name.required"
```

Comment:
Here things are gonna be a little bit different. Something like:

```slim
span.error-messages v-if="errors.name"
  span.error-message v-for="(error, index) in errors.name" :key="index"
    | {{ error }}
```

This will require some styling/positioning fix to break line when multiple errors happen. We can handle that when we reach that state.